src/IECoreScene/CurvesAlgo.cpp

```diff
@@ -490,25 +369,31 @@ CurvesPrimitivePtr deleteCurves( const CurvesPrimitive *curvesPrimitive, const t
 			case PrimitiveVariable::Uniform:
 			{
 				IECore::Data *inputData = const_cast< IECore::Data * >( it->second.data.get() );
```

Comment:
Can we remove this `const_cast` now? I'm hoping it was just there because `despatchTypedData()` didn't have a const variant.